lib/git-controller.js

```diff
@@ -20,4 +20,13 @@ async function getOrgProjects(org) {
   return await request(options);
 }
 
-export { getOrgUsers, getOrgProjects };
+async function getProjectUsers(owner, repo) {
+  let options = {
+    method: 'GET',
+    url: `https://api.github.com/repos/${owner}/${repo}/contributors`,
+    headers: {'user-agent': 'node.js'}
+  };
+  return await request(options);
```

Comment:
it would be pretty cool if you can write a function say 
```
async function makeGitApiReqeust(method, api) {
.....
}
```
and use this here and else where so we can abstract some boilerplate code